Update cors origin default info (#49489)
quarkusio/quarkus: docs/src/main/asciidoc/security-cors.adoc

----

<1> Enables the CORS filter.

> <2> Specifies allowed origins, including a regular expression. Leaving this value empty blocks all origins.

Reviewer: I don't mind having the information here but is it also in the config doc for the property?

> <2> Specifies allowed origins, including a regular expression. When not specified, CORS headers are not sent.

Reviewer 2: ```suggestion
<2> Specifies allowed origins, including a regular expression. When not specified, CORS is not permitted.
```

There is another property for controlling the extra headers, so it might be a bit confusing
